src/main/java/Avocado.java

```diff
@@ -0,0 +1,31 @@
+import java.util.Scanner;
+public class Avocado {
+    public static void main(String[] args) {
+        System.out.println("Hello from avocado");
+        System.out.println("How can I help you ?");
+        System.out.println("See you!");
+        String[] list = new String[100];
+        int i = 0;
+        int n = 0;
```

Comment:
It would be better to give a more descriptive name to the variables, for example, `taskNumber` instead of `n`